lib/offline/db.ts

```diff
@@ -21,8 +23,8 @@ export class OvoflowDexie extends Dexie {
 
   constructor() {
     super(DATABASE_NAME);
-    this.version(1).stores({
-      technicianVisit: '++id, value, farmerId',
+    this.version(2).stores({
```

Comment:
Could we make the version number a constant?

Reply:
Yeah, I can do that